sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -242,6 +251,16 @@ private[client] class Shim_v0_12 extends Shim with Logging {
       JInteger.TYPE,
       JBoolean.TYPE,
       JBoolean.TYPE)
+  private lazy val moveFileMethod =
+    findMethod(
+      classOf[Hive],
+      "moveFile",
```

Comment:
`moveFile` is supported from  [Hive-1.1.0](https://issues.apache.org/jira/browse/HIVE-9264).